packages/flutter_tools/bin/podhelper.rb

```diff
@@ -32,7 +32,7 @@ def flutter_additional_ios_build_settings(target)
   return unless target.platform_name == :ios
 
   # [target.deployment_target] is a [String] formatted as "8.0".
-  inherit_deployment_target = target.deployment_target[/\d+/].to_i < 11
+  inherit_deployment_target = target.deployment_target[/\d+/].to_i < 12
```

Comment:
Everything before this is from the auto-migrate, this is the start of the real tool change.